### PR TITLE
Update duration page and change duration-instant to 0ms

### LINKED ISSUE #1828: Foundations: Finalise Duration doc

### PR DESCRIPTION
**Pull request by james-nash**
Fleshes out the duration foundation page.

Direct link to preview page: https://saltdesignsystem-git-foundations-duration-joshwooding.vercel.app/salt/foundations/duration

Closes #1828 

Changes duration-instant token to 0ms @pseys

### CODE REVIEW

**Comment by joshwooding** [comment]
This PR has been open without any updates, can it be closed?

**Comment by james-nash** [comment]
Can we keep this PR please. @pseys and I set this up so that @navkaur76 can work on the duration page (I guess she's just not had a chance to get started yet). The branch + PR allows folks like her to edit the page MDX and see the results via a preview build all via GitHub's web UI.

**Comment by pseys** [comment]
> Can we keep this PR please. @pseys and I set this up so that @navkaur76 can work on the duration page (I guess she's just not had a chance to get started yet). The branch + PR allows folks like her to edit the page MDX and see the results via a preview build all via GitHub's web UI.

I can second that. We're reviewing the content at the moment so should be able to update it soon.

File: site/docs/foundations/duration.mdx [hunk 1]
```diff
@@ -20,21 +18,78 @@ There are four durations available to choose from: Instant, Perceptible, Notable
...
 
-With a notable delay, the user is aware the system is slow to respond. A feeling of loss of control may occur, which is why it's important the delay is clearly communicated. Notable delays may be required in order to load a data grid or retrieve search results.
+<Image
+  src="/img/foundations/duration-perceptible.svg"
+  alt="Example of duration perceptible"
+/>
 
-### Cut-off
+Perceptible is recommended for when you want to reveal secondary guidance or information to your user, but it doesn't warrant a permanent place within the interface. Content typically has a relatively high level of urgency, so that users can quickly find solutions to their problems and avoid getting frustrated with the system.
```
Comment (alycrys) [inline site/docs/foundations/duration.mdx:39]:
```suggestion
Perceptible duration is recommended for when you want to reveal secondary guidance or information to your user, but it doesn't warrant a permanent place within the interface. Content typically has a relatively high level of urgency, so that users can quickly find solutions to their problems and avoid getting frustrated with the system.
```

File: site/public/img/foundations/duration-perceptible.svg
Comment (joshwooding) [inline site/public/img/foundations/duration-perceptible.svg:1]: Tooltip looks weird here

File: site/docs/foundations/duration.mdx [hunk 2]
```diff
@@ -20,21 +18,78 @@ There are four durations available to choose from: Instant, Perceptible, Notable
 | `--salt-duration-notable`     | 1000                    |
 | `--salt-duration-cutoff`      | 10000                   |
 
-### Instant
+## Instant (0-100ms)
```
Comment (libertymayc) [inline site/docs/foundations/duration.mdx:21]: If we do change this to 0ms, do we still want to say 100ms in this section? @pseys

Comment (libertymayc) [reply site/docs/foundations/duration.mdx:21]: I can also add the update into this PR if we confirm changing it to 0

File: site/docs/foundations/duration.mdx [hunk 3]
```diff
@@ -20,21 +18,78 @@ There are four durations available to choose from: Instant, Perceptible, Notable
...
+
+To prevent a frustrating or unproductive user experience, the time up until a cut-off duration can be used for:
+
+- Displaying messages that inform the user of the current interaction and which processes are taking time.
+- Displaying general hints and tips to using the application.
+- If possible, allowing the user to continue using other parts of the UI whilst the content loads.
+
+#### By-passing the 10 second cut-off point:
+
+There may be a requirement for a system to continue loading an interaction for an indeterminate amount of time, past the cut-off point, due to a workflow dependancy. A system may need to support a highly performant UI containing significant amounts of data points concurrently updating on a UI.
```
Comment (Fercas123) [inline site/docs/foundations/duration.mdx:84]:
```suggestion
There may be a requirement for a system to continue loading an interaction for an indeterminate amount of time, past the cut-off point, due to a workflow dependency. A system may need to support a highly performant UI containing significant amounts of data points concurrently updating on a UI.
```

File: site/docs/foundations/accent.mdx
Comment (joshwooding) [inline site/docs/foundations/accent.mdx:1]: Is this change intentional? It's not related to duration

Comment (navkaur76) [reply site/docs/foundations/accent.mdx:1]: Not intentional, Paul created this when demo-ing MDX to the design team. It can be removed from duration

File: site/public/img/foundations/duration-perceptible.svg
Comment (navkaur76) [reply site/public/img/foundations/duration-perceptible.svg:1]: I'll update the image, thanks!

File: site/docs/foundations/duration.mdx (hunk 2 above)
Comment (libertymayc) [reply site/docs/foundations/duration.mdx:21]: I made the change in this PR, cc @navkaur76 @pseys

File: site/docs/foundations/duration.mdx [hunk 4]
```diff
@@ -13,28 +11,93 @@ The time between the moment a user performs an action and the moment feedback is
...
+  alt="Example of duration instant"
+/>
+
+## Perceptible (300ms)
+
+A perceptible duration is a short, but detectable delay between a user's action and the system's response. The user should be aware of the gap but retain the feeling of control. This implies a shared control between the user and the interface itself—the user expects the system to process the interaction they have enabled the system to do. However, the delay should feel acceptable, and therefore maintain the user’s attention and productivity. This aligns with the “[Doherty Threshold](https://lawsofux.com/doherty-threshold/)” principle which dictates that a user experience turns from painful to addictive after the system feedback time drops below 400ms. Tooltips and overlays are examples of perceptible delays.
+
+<Image
+  src="/img/foundations/duration-perceptible.svg"
+  alt="Example of duration perceptible"
```
Comment (libertymayc) [inline site/docs/foundations/duration.mdx:42]:
```suggestion
  alt="Example of duration perceptible"  
  className="saltSite-util-space-below-1x"
```

File: site/docs/foundations/accent.mdx
Comment (james-nash) [reply site/docs/foundations/accent.mdx:1]: I've removed this file now.

File: site/docs/foundations/index.mdx [hunk 5]
```diff
@@ -18,7 +18,17 @@ Foundations are the fundamental principles that guide the creation of a cohesive
 Density relates to the amount of content that can fit on a screen based on the size and spacing of components.
 
   </OverviewCard>
-  
+
+<OverviewCard
+    icon={<Image src="/img/foundations/duration.svg" alt="duration"  />}
+    title="Duration"
+    url="./duration"
+    keylineColor="teal"
```
Comment (JoanaMMoreira) [inline site/docs/foundations/index.mdx:26]: Do we want this to be the same colour as the Density card?

**Review by JoanaMMoreira — DISMISSED** [review]

**Review by JoanaMMoreira — APPROVED** [review]

**Review by origami-z — APPROVED** [review]
a little confused when seeing instant token as `0` in table and `0-100` in heading after

**Review by pseys — APPROVED** [review]
